native_client/java/Makefile

```diff
@@ -20,12 +22,50 @@ libdeepspeech/libs/%/libdeepspeech.so:
 	-mkdir libdeepspeech/libs/$*/
 	cp ${TFDIR}/bazel-out/$*-*/bin/native_client/libdeepspeech.so libdeepspeech/libs/$*/
 
-apk: apk-clean bindings $(patsubst %,libdeepspeech/libs/%/libdeepspeech.so,$(ARCHS))
+apk-prepare:
```

Comment:
this is weird and likely error prone, we should rather have `libdeepspeech_android/build.gradle` and use that directly instead of doing this `cp` dance